woocommerce/woocommerce-ios
# [Woo POS] [Cash & Receipts] Modal view to send receipts after successful payment

Part of #14459
Closes: #14461 

## Description

This PR adds a modal view that will be triggered when tapping on the "Receipt" button, after a successful POS transaction. The modal contains a simple textfield where we'll enter the customer email address, so we can send an email receipt. Designs and error handling are TBD.

https://github.com/user-attachments/assets/e980008b-542f-46a2-bc17-0a1c107d2631

## Testing information
- Set feature flag `.sendReceiptsForPointOfSale`  to `true`
- Log into Povilas testing site https://lovely-aware-sturgeon.jurassic.ninja/ details: PdfdoF-5Qj-p2
- Proceed with a POS transaction.
- Tap on the textfield for focus, observe the built-in keyboard can be opened (cmd+K in simulator)
- After entering a valid email address, tap on "send", check your email and confirm the email has been received.
- The modal should be dismissed automatically upon completion (it may take 2-3 seconds due to the multiple network requests that are currently necessary). UI while the user waits will be improved in following PRs.

---

## Reviewer (or Author, in the case of optional code reviews):

Please make sure these conditions are met before approving the PR, or request changes if the PR needs improvement:
